lib/redlock/client.rb

```diff
@@ -272,7 +296,12 @@ def lock_instances(resource, ttl, options)
         { validity: validity, resource: resource, value: value }
       else
         @servers.each { |s| s.unlock(resource, value) }
-        false
+
+        if errors.size >= @quorum
+          raise LockAcquisitionError.new("Too many Redis errors prevented lock acquisition", errors)
```

Comment:
if there's no string interpolation, then single quotes are preferred:
```suggestion
          raise LockAcquisitionError.new('Too many Redis errors prevented lock acquisition', errors)
```